reward/stakingInfo_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package reward
+
+import (
+	"github.com/klaytn/klaytn/common"
+	"github.com/stretchr/testify/assert"
+	"math"
+	"testing"
+)
+
+func TestStakingInfo_GetIndexByNodeId(t *testing.T) {
+	testdata := []common.Address{
+		common.StringToAddress("0xB55e5986b972Be438b4A91d6e8726aA50AD55EDc"),
+		common.StringToAddress("0xaDfc427080B4a66b5a629cd633d48C5d734572cA"),
+		common.StringToAddress("0x994daB8EB6f3FaE044cC0c9a0AB1A038e136b0B6"),
+		common.StringToAddress("0xD527822212Fded72c5fE89f46281d5355BD58235"),
+	}
+	testCases := []struct {
+		address common.Address
+		index   int
+	}{
+		{common.StringToAddress("0xB55e5986b972Be438b4A91d6e8726aA50AD55EDc"), 0},
+		{common.StringToAddress("0xaDfc427080B4a66b5a629cd633d48C5d734572cA"), 1},
+		{common.StringToAddress("0x994daB8EB6f3FaE044cC0c9a0AB1A038e136b0B6"), 2},
+		{common.StringToAddress("0xD527822212Fded72c5fE89f46281d5355BD58235"), 3},
+	}
+
+	stakingInfo, error := newEmptyStakingInfo(0)
+	if error != nil {
+		t.Errorf("error has occurred. error : %v", error)
+		t.FailNow()
+	}
+	stakingInfo.CouncilNodeIds = testdata
+
+	for i := 0; i < len(testCases); i++ {
+		result := stakingInfo.GetIndexByNodeId(testCases[i].address)
+		if result == AddrNotFoundInCouncilNodes {
+			t.Errorf("Address is not found in stakingInfo")
+		}
+		assert.Equal(t, testCases[i].index, result)
+	}
```

Comment:
How about adding exception case like returing `AddrNotFoundInCouncilNodes` with wrong address for code coverage?

Reply:
thanks for your comment. I've reflected it